### ISSUE TITLE
Crash on opening a malformed .h5 file

### ISSUE THREAD

**Issue by greyltc** [mutovis/analysis-software#7]
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/gui.py", line 1182, in handleWatchAction
    self.handleWatchUpdate(self.workingDirectory)
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/gui.py", line 1233, in handleWatchUpdate
    self.newFiles(new_files_fp)
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/gui.py", line 899, in newFiles
    self.analyzer.processFiles(fullPaths, self.processFitResult, self.primeRow)
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/ivAnalyzer.py", line 305, in processFiles
    fileDatas = ivAnalyzer._loadFile(fullPath)
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/ivAnalyzer.py", line 374, in _loadFile
    iv_data = pixel['all_measurements']          
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/usr/lib/python3.7/site-packages/h5py/_hl/group.py", line 262, in __getitem__
    oid = h5o.open(self.id, self._e(name), lapl=self._lapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5o.pyx", line 190, in h5py.h5o.open
KeyError: "Unable to open object (object 'all_measurements' doesn't exist)"
Aborted (core dumped)
```
This file doesn't contain the `all_measurements` dataset and so things don't go well. Originally reported in https://github.com/mutovis/control-software/issues/2#issuecomment-468447573

**Comment by greyltc** [comment]
@rbrenesh, this should be solved in c7250587471693329679c656275884bbbb753721 which i rolled into the v3.03 release (`yay -S python-mutovis-analysis`)

**Comment by rbrenesh** [comment]
Nice, I'll try it out in a few hours. Do you still need me to send you the file?
